dashboards/kubernetes/metadata.yaml

```diff
@@ -1,4 +1,12 @@
 sample_dashboards:
+  -
+    category: Kubernetes
+    id: k8s-overview-prometheus
+    display_name: Kubernetes Infrastructure Overview
```

Comment:
```suggestion
    display_name: Kubernetes Infrastructure Prometheus Overview
```

Reply:
Updated all display names to include "Prometheus", as well as the README dashboard titles.